content/handbook/support/workflows/how-to-create-npm-package.md

```
@@ -0,0 +1,183 @@
+---
+title: NPM GitLab レジストリの利用
+upstream_path: /handbook/support/workflows/how-to-create-npm-package/
+upstream_sha: 47fdb6582389288bed0f04a23aa5d972c3ce1ff5
+translated_at: "2026-05-08T00:00:00Z"
+translator: claude
+stale: false
+---
+
+## 概要
+
+このワークフローの目的は、パッケージにアクセスできるようにすることで、よくある顧客のシナリオを簡単（あるいはより簡単）にテストできるようにすることです。以下に示す手順に従ってパッケージを作成するか、[こちら](https://gitlab.com/sahbabou/test-package)を利用できます。パッケージを公開する際は、正しいプロジェクト ID を参照することを忘れないでください [(ステップ 3)](#3-publishing-the-scoped-package)。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)